org/eclipse/persistence/org.eclipse.persistence.moxy/2.6.5.payara-p1/org.eclipse.persistence.moxy-2.6.5.payara-p1.pom

```diff
@@ -0,0 +1,68 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.eclipse.persistence</groupId>
+  <artifactId>org.eclipse.persistence.oracle</artifactId>
```

Comment:
You're adding the artifact to the wrong location - this is in persistence.moxy not persistence.oracle